### PR TITLE
Issue/16

### PR DESCRIPTION
**Pull request by growdev**
I'm working toward getting full coverage before this is merged. Fix for #16.

Current unit test output:

```
...SSS.....SSSSSSSSSSSSSSSSSSSS                                   31 / 31 (100%)

Time: 1.79 seconds, Memory: 38.75MB

OK, but incomplete, skipped, or risky tests!
Tests: 31, Assertions: 15, Skipped: 23.
```

### CODE REVIEW

File: tests/phpunit/factories/class-sugar-factory-for-events.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+		update_post_meta( $post_id, 'sc_event_end_date_time', $final_end_date_time );
+
+		return $post_id;
+	}
+
+	/**
+	 * @param $args
+	 */
+	function create_recurring( $args ) {
+		// TODO: Finish when needed
```
Comment (pippinsplugins) [inline tests/phpunit/factories/class-sugar-factory-for-events.php]: Could you add this please?

File: tests/phpunit/factories/class-sugar-factory-for-events.php [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+	/**
+	 *
+	 * @param int $event_id
+	 * @param array $args
+	 *
+	 * @return bool
+	 */
+	function update_object( $event_id, $args ) {
+		// TODO: Finish when needed
```
Comment (pippinsplugins) [inline tests/phpunit/factories/class-sugar-factory-for-events.php]: Anything necessary to do here?

File: tests/test-functions.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+		$month_name = sc_month_num_to_name( 12 );
+		$this->assertEquals( 'December', $month_name );
+
+	}
+
+	function test_sc_is_calendar_page() {
+
+		$is_calendar_page = sc_is_calendar_page();
+		$this->assertFalse( $is_calendar_page );
+
```
Comment (pippinsplugins) [inline tests/test-functions.php]: Could we get a full test added here?

File: tests/test-functions.php [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+		// TODO: setup calendar page an switch to it
+		//$is_calendar_page = sc_is_calendar_page();
+		//$this->assertTrue( $is_calendar_page );
+
+	}
+
+	function test_sc_get_event_date() {
+
```
Comment (pippinsplugins) [inline tests/test-functions.php]: This needs finished :)

File: tests/test-list-table-columns.php [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+		$typenow = 'sc_event';
+
+		ob_start();
+		sc_add_event_filters();
+		$output = ob_get_clean();
+
+		$this->assertTrue( empty( $output ) );
+
+		//TODO  Finish this when a category factory has been added.
+		/*
```
Comment (pippinsplugins) [inline tests/test-list-table-columns.php]: Let's get this finished.

File: tests/phpunit/factories/class-sugar-factory-for-events.php (hunk 1 above)
Comment (growdev) [inline tests/phpunit/factories/class-sugar-factory-for-events.php]: This is dependent on code from #64.

File: tests/phpunit/factories/class-sugar-factory-for-events.php (hunk 2 above)
Comment (growdev) [inline tests/phpunit/factories/class-sugar-factory-for-events.php]: Dependent on #64.

File: tests/test-functions.php (hunk 3 above)
Comment (growdev) [inline tests/test-functions.php]: Will do!

File: tests/phpunit/factories/class-sugar-factory-for-events.php (hunk 2 above)
Comment (growdev) [inline tests/phpunit/factories/class-sugar-factory-for-events.php]: Actually, removed this as it's not needed.

File: tests/phpunit/factories/class-sugar-factory-for-events.php (hunk 1 above)
Comment (growdev) [inline tests/phpunit/factories/class-sugar-factory-for-events.php]: Added.

File: tests/test-list-table-columns.php (hunk 5 above)
Comment (growdev) [inline tests/test-list-table-columns.php]: Finished.

**Comment by growdev** [comment]
@pippinsplugins why is Travis running tests against the target of the merge `release/1.6.0` ?

**Comment by pippinsplugins** [comment]
Because that's the branch the changes are getting merged into.

File: tests/test-functions.php (hunk 3 above)
Comment (growdev) [inline tests/test-functions.php]: Done

File: tests/test-functions.php (hunk 4 above)
Comment (growdev) [inline tests/test-functions.php]: Finished

**Pull request merged**